src/main/java/com/alkemy/ong/application/rest/request/NewsRequest.java

```diff
@@ -0,0 +1,30 @@
+package com.alkemy.ong.application.rest.request;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+public class NewsRequest {
+
+  @Size(max = 50)
+  @Pattern(regexp = "[a-zA-Z\\s]+", message = "name must contain letters and blank spaces")
+  @NotEmpty(message = "name may not be empty")
+  private String name;
+
+  @Pattern(regexp = "[a-zA-Z0-9\\s]+", message = "text must contain letters and numbers")
```

Comment:
Esta expresión regular no me estaría funcionando. Creo que lo correcto seria: ^[a-zA-Z_ ]*$

Adicionalmente, el mensaje debería ser: "The text has invalid format." o "The text accepts only alphanumeric characters and blank spaces."

FYI: testeado en https://regex101.com/